sources/assets/Stride.Core.Assets.CompilerApp/Stride.Core.Assets.CompilerApp.csproj

```diff
@@ -14,7 +14,7 @@
     <SchemaVersion>2.0</SchemaVersion>
     <OutputType>Exe</OutputType>
     <StrideAssemblyProcessor>true</StrideAssemblyProcessor>
-    <TargetFramework>$(StrideEditorTargetFramework)</TargetFramework>
+    <TargetFramework>$(StrideXplatEditorTargetFramework)</TargetFramework>
```

Comment:
Did you managed to run and test compilerapp on net target ? I had difficulties in the past 😅

Reply:
I'll check.

Reply:
Seems fine:
![image](https://github.com/stride3d/stride/assets/3006525/3ecb457c-9651-4cdb-b080-bc72d06443c8)